framework/hpp_api_vulkan_sample.h

```diff
@@ -60,7 +60,7 @@ struct HPPVertex
  *
  * See vkb::ApiVulkanSample for documentation
  */
-class HPPApiVulkanSample : public vkb::HPPVulkanSample
+class HPPApiVulkanSample : public vkb::VulkanSample<true>
 {
```

Comment:
I typically rely on a convention like this..

```suggestion
class HPPApiVulkanSample : public vkb::VulkanSample<true>
{
    using Parent = vkb::VulkanSample<true>;
```

so that in my code I can always write `Parent::parent_method()` instead of having to write the particular base class out each time.

Reply:
Great approach!
I think, we should introduce such a Parent (or ParentType?). But with subsequent PRs.